warehouse/templates/manage/account/totp-provision.html

```diff
@@ -0,0 +1,67 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "manage/manage_base.html" %}
+
+{% set user = request.user %}
+{% set title = "Set up two factor authentication using an application (TOTP)" %}
+
+{% block title %}{{ title }}{% endblock %}
+
+{# Hide mobile search on manager pages #}
+{% block mobile_search %}{% endblock %}
+
+{% block main %}
+<h1 class="page-title">{{ title }}</h1>
+<section>
+  <h2>How 2FA with TOTP works</h2>
+
+  <ol>
+    <li>When you sign into PyPI, you will be asked to provide your username and password, as usual</li>
+    <li>You will then see a screen asking for an additional authentication code</li>
+    <li>Use the application of your choice (provisioned below) to generate a code</li>
+    <li>Enter the code to access your PyPI account</li>
+  </ol>
+
+  <h2>Provisioning your application</h2>
+
+  <p>You can provision any compatible TOTP application to generate two factor authentication codes for PyPI. We recommend <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_GB">Google Authenticator for Android</a> or <a href="https://itunes.apple.com/gb/app/google-authenticator/id388497605?mt=8">iPhone</a>, <a href="https://support.1password.com/one-time-passwords/">1Password</a>, <a href="https://authy.com/">Authy</a> or <a href="https://lastpass.com/auth/">LastPass Authenticator</a>.</p>
```

Comment:
Perhaps we could call out [Duo Mobile](https://duo.com/product/trusted-users/two-factor-authentication/authentication-methods/totp) here.

Also, perhaps we could whittle this down to one or two suggestions. A laundry list of options will potentially confuse and overwhelm users.

Reply:
Actually, this would be a better page to link to: https://guide.duo.com/third-party-accounts

Reply:
I like this idea @mschwager, I agree that too many options could confuse some users. Google Authenticator and Duo seem like a good, minimal set of recommendations to me.

Reply:
I agree!

Reply:
@nlhkabu you originally wrote this part of the page -- any thoughts?

Reply:
May I ask adding FreeOTP/FreeOTP+ to the list? The reason for this is that they are open source and others are AFAIK proprietary which makes them questionable from the security recommendations PoV.